### ISSUE TITLE
Command 'Convert YAML files to JSON' resulted in an error (Cannot destructure property 'fsPath' of 't' as it is undefined.)

### ISSUE THREAD

**Issue by singularitti** [hilleer/vscode-yaml-plus-json#13]
When I am try to convert a JSON file to a YAML file, but running `Convert JSON files to YAML` from command palette, it tells me:
```
Command 'Convert JSON files to YAML' resulted in an error (command 'extension.convertJsonFilesToYaml' not found)
onDidChangeNotification @ /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:2855
```

When I select the content of that file and run `Convert selection to YAML` from command palette, it tells me:
```
Command 'Convert selection to YAML' resulted in an error (command 'extension.jsonSelectionToYaml' not found)
onDidChangeNotification @ notificationsAlerts.ts:40
(anonymous) @ notificationsAlerts.ts:26
fire @ event.ts:587
addNotification @ notifications.ts:207
error @ notificationService.ts:59
accept @ commandsQuickAccess.ts:162
async function (async)
accept @ commandsQuickAccess.ts:159
(anonymous) @ pickerQuickAccess.ts:263
fire @ event.ts:587
(anonymous) @ quickInput.ts:722
fire @ event.ts:587
(anonymous) @ quickInput.ts:1210
```

When I run `Convert to YAML`, it tells me
```
  | onDidChangeNotification | @ | notificationsAlerts.ts:40
-- | -- | -- | --
  | (anonymous) | @ | notificationsAlerts.ts:26
  | fire | @ | event.ts:587
  | addNotification | @ | notifications.ts:207
  | error | @ | notificationService.ts:59
  | accept | @ | commandsQuickAccess.ts:162
  | async function (async) |   |  
  | accept | @ | commandsQuickAccess.ts:159
  | (anonymous) | @ | pickerQuickAccess.ts:263
  | fire | @ | event.ts:587
  | (anonymous) | @ | quickInput.ts:722
  | fire | @ | event.ts:587
  | (anonymous) | @ | quickInput.ts:1210
```
![image](https://user-images.githubusercontent.com/25192197/79064822-15b10b00-7c7a-11ea-8fcb-739a63163d14.png)

But if I select that file in the file explorer and run from right selection of `Convert to YAML`, it works. Renaming the extension to `.yaml` works, too.

VSCode version:
```
Version: 1.44.0
Commit: 2aae1f26c72891c399f860409176fe435a154b13
Date: 2020-04-08T08:23:56.137Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.4.0
```
Plugin version: 1.2.0

**Comment by hilleer** [comment]
Hi @singularitti,

Thanks a lot for reporting this. I will take a look into it.

**Comment by hilleer** [comment]
Hi again,

I published version `1.2.1` that now hides following commands from the command palette:

* `Convert JSON files to YAML`
* `Convert YAML files to JSON`
* `Convert selected files to YAML`
* `Convert selected files to JSON`

For commands `Convert selection to YAML` and `Convert to YAML` can you try to:

1. Update to latest extension version.
2. Reload vscode window (update would _normally_ require a reload).
3. Try again.

If you still have issues please do not hesitate to comment here again. Closing for the time being.
